packages/react-router/modules/__tests__/Router-test.js

```diff
@@ -50,17 +51,14 @@ describe("A <Router>", () => {
 
   describe("context", () => {
```

Comment:
Let's go ahead and remove all tests that test our context values in this PR. Since context is not part of our public API, we shouldn't have them.